src/clib/lib/enkf/ecl_config.cpp

```diff
@@ -1,483 +0,0 @@
-            "Could not load summary case from:%s \n", refcase);
-        ui_return_add_error(ui_return, error_msg);
-        free(error_msg);
-        return ui_return;
-    }
-}
-
-/**
- Will return NULL if no refcase is set.
-*/
-const char *ecl_config_get_refcase_name(const ecl_config_type *ecl_config) {
-    auto refcase = ecl_config->refcase;
-    if (!refcase)
-        return NULL;
-    return ecl_sum_get_case(ecl_config->refcase);
-}
-
-static ecl_config_type *ecl_config_alloc_empty(void) {
-    ecl_config_type *ecl_config = new ecl_config_type();
-
-    ecl_config->io_config = ecl_io_config_alloc(
-        DEFAULT_FORMATTED, DEFAULT_UNIFIED, DEFAULT_UNIFIED);
-    ecl_config->have_eclbase = false;
-    ecl_config->num_cpu =
-        1; /* This must get a valid default in case no ECLIPSE datafile is provided. */
-    ecl_config->unit_system = ECL_METRIC_UNITS;
-    ecl_config->data_file = NULL;
-    ecl_config->grid = NULL;
-    ecl_config->can_restart = false;
-    ecl_config->schedule_prediction_file = NULL;
-    ecl_config->refcase = NULL;
-
-    return ecl_config;
-}
-
-ecl_config_type *ecl_config_alloc(const config_content_type *config_content) {
-    ecl_config_type *ecl_config = ecl_config_alloc_empty();
-
-    if (config_content)
-        ecl_config_init(ecl_config, config_content);
-
-    return ecl_config;
-}
-
-ecl_config_type *ecl_config_alloc_full(bool have_eclbase, char *data_file,
-                                       ecl_grid_type *grid,
-                                       char *refcase_default,
-                                       char *sched_prediction_file) {
-    ecl_config_type *ecl_config = ecl_config_alloc_empty();
-    ecl_config->have_eclbase = have_eclbase;
-    ecl_config->grid = grid;
-    if (data_file != NULL) {
-        ecl_config_set_data_file(ecl_config, data_file);
-    }
-
-    if (refcase_default)
-        if (!ecl_config_load_refcase(ecl_config, refcase_default))
-            fprintf(stderr, "** Warning: loading refcase:%s failed \n",
-                    refcase_default);
-
-    if (sched_prediction_file)
-        ecl_config->schedule_prediction_file =
-            util_alloc_string_copy(sched_prediction_file);
-
-    return ecl_config;
-}
-
-static void handle_has_eclbase_key(ecl_config_type *ecl_config,
-                                   const config_content_type *config) {
-    /*
-     The eclbase is not internalized here; here we only flag that the
-     ECLBASE keyword has been present in the configuration. The
-     actualt value is internalized as a job_name in the model_config.
-   */
-
-    if (config_content_has_item(config, ECLBASE_KEY)) {
-        ui_return_type *ui_return = ecl_config_validate_eclbase(
-            ecl_config, config_content_iget(config, ECLBASE_KEY, 0, 0));
-        if (ui_return_get_status(ui_return) == UI_RETURN_OK)
-            ecl_config->have_eclbase = true;
-        else
-            util_abort("%s: failed to set eclbase format. Error:%s\n", __func__,
```

Comment:
Abort is bad, but we should probably have similar error messages raised.

Reply:
eclbase is no longer used, so we can scrap the warning, too!